[mgmt] workloadssapvirtualinstance release (#32424)
Azure/azure-sdk-for-js: sdk/workloads/arm-workloadssapvirtualinstance/CHANGELOG.md

# Release History
    
## 1.0.0 (2025-01-06)

### Features Added

> The package of @azure/arm-workloadssapvirtualinstance is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/azsdk/js/mgmt/quickstart).

Author: ```suggestion
This is the first stable version with the package of @azure/arm-workloadssapvirtualinstance
```
